vendor/bat-native-confirmations/src/bat/confirmations/internal/confirmation_info.cc

```diff
@@ -7,28 +7,26 @@
 
 namespace confirmations {
 
-ConfirmationInfo::ConfirmationInfo() :
-    id(""),
-    creative_instance_id(""),
-    type(ConfirmationType::UNKNOWN),
-    token_info(TokenInfo()),
-    payment_token(nullptr),
-    blinded_payment_token(nullptr),
-    credential(""),
-    timestamp_in_seconds(0),
-    created(false) {}
+ConfirmationInfo::ConfirmationInfo()
+    : type(ConfirmationType::UNKNOWN),
+      payment_token(nullptr),
+      blinded_payment_token(nullptr),
+      timestamp_in_seconds(0),
+      created(false) {}
 
-ConfirmationInfo::ConfirmationInfo(const ConfirmationInfo& info) :
-    id(info.id),
-    creative_instance_id(info.creative_instance_id),
-    type(info.type),
-    token_info(info.token_info),
-    payment_token(info.payment_token),
-    blinded_payment_token(info.blinded_payment_token),
-    credential(info.credential),
-    timestamp_in_seconds(info.timestamp_in_seconds),
-    created(info.created) {}
+ConfirmationInfo::ConfirmationInfo(
+    const ConfirmationInfo& info)
+    : id(info.id),
+      creative_instance_id(info.creative_instance_id),
+      type(info.type),
+      token_info(info.token_info),
+      payment_token(info.payment_token),
+      blinded_payment_token(info.blinded_payment_token),
+      credential(info.credential),
+      timestamp_in_seconds(info.timestamp_in_seconds),
+      created(info.created) {
+}
```

Comment:
Could this just be defaulted with `= default'?

Reply:
Unfortunately not due to `Token` and `BlindedToken`

Reply:
actually it can, so fixing now

Reply:
Fixed